### DISCUSSION TITLE
GitHub Support is ignoring me - 2 Months of waiting

### DISCUSSION THREAD

**Discussion by Sa-Cl** [community/community#126220] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello Everyone. 

On 25 of March GitHub suspended my account without any reason or email notice. I contacted them and until today (28 of Mai) they did not answered. One month ago my friend wrote in this community a post referring to this problem, and the maintainer closed the thread with the response : "This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours."

Right now, after 2 months it's hard to believe that GitHub is going to answer or to investigate. What can I do besides wait? Because trust me, I'm sick of waiting.

**Comment by samarjitsahoo** [comment]
Well lets wait for the correct answers!

**Comment by rlzyy** [comment]
Just badges

**Comment by ghostinhershell** [comment]
Hi @Sa-Cl,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
